web/src/components/tasks/business-formation/business/MainBusinessAddressNj.tsx

```diff
@@ -61,6 +64,11 @@ export const MainBusinessAddressNj = (): ReactElement => {
       </div>
       {isExpanded && (
         <>
+          {industryId === "cannabis" && (
+            <Alert variant="warning">
+              <Content>{Config.profileDefaults.default.cannabisLocationAlert}</Content>
+            </Alert>
+          )}
```

Comment:
This 5-line code chunk is duplicated a bunch and could be its own component

Reply:
included